infobip_channels/web_rtc/channel.py

```diff
@@ -0,0 +1,158 @@
+from http import HTTPStatus
+from typing import Any, Dict, Type, Union
+
+import requests
+
+from infobip_channels.core.channel import Channel
+from infobip_channels.core.models import ResponseBase
+from infobip_channels.web_rtc.models.body.generate_token import GenerateTokenBody
+from infobip_channels.web_rtc.models.body.save_application import SaveApplicationBody
+from infobip_channels.web_rtc.models.body.update_application import (
+    UpdateApplicationBody,
+)
+from infobip_channels.web_rtc.models.path_parameters.core import PathParameter
+from infobip_channels.web_rtc.models.path_parameters.web_rtc_application import (
+    WebRtcPathParameters,
+)
+from infobip_channels.web_rtc.models.response.core import (
+    WebRtcResponseError,
+    WebRtcResponseOK,
+)
+from infobip_channels.web_rtc.models.response.generate_token import (
+    GenerateTokenResponseOK,
+)
+from infobip_channels.web_rtc.models.response.get_applications import (
+    GetApplicationsResponseOK,
+)
+
+
+class WebRtcChannel(Channel):
+    """Class used for interaction with the Infobip's WhatsApp API."""
+
+    WEB_RTC_URL_TEMPLATE = "/webrtc/1/"
+
+    @staticmethod
+    def validate_path_parameter(
```

Comment:
We should extract this into the base `Channel` class since it's used here and in the `WhatsAppChannel` class. Then we can remove the method from this class and the `WhatsAppChannel` one.

Reply:
Top, done! That is clean 👍🏻